src/sql/workbench/contrib/assessment/browser/consts.ts

```diff
@@ -0,0 +1,16 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export enum AssessmentTargetType {
```

Comment:
These should go in common instead - they don't need browser apis

Reply:
Moved to /common